WEB/Src/DependencyCollector/DependencyCollector/Implementation/AzureSdk/AzureSdkDiagnosticListenerSubscriber.cs

```diff
@@ -27,7 +30,7 @@ public override void Dispose()
 
         internal override bool IsSourceEnabled(DiagnosticListener diagnosticListener)
         {
-            return diagnosticListener.Name.StartsWith(DiagnosticListenerName, StringComparison.Ordinal);
+            return diagnosticListener.Name.StartsWith(DiagnosticListenerName, StringComparison.Ordinal) && !diagnosticListener.Name.Equals(CosmosRequestSourceName);
```

Comment:
Please make one change here.
Use `StringComparison.Ordinal` in the `Equals` method to clear the build warnings.

Reply:
sure, done!